cmd/shield/import.go

```diff
@@ -160,6 +162,9 @@ func (m *ImportManifest) Normalize() error {
 				if job.Retain == "" {
 					return fmt.Errorf("Tenant '%s', data system '%s', job '%s' is missing its `retain' attribute", tenant.Name, system.Name, job.Name)
 				}
+				if job.Retries == 0 {
+					return fmt.Errorf("Tenant '%s', data system '%s', job '%s' is missing its `retries' attribute", tenant.Name, system.Name, job.Name)
```

Comment:
Will this cause a problem for someone who upgraded with active jobs in play that will default to `0` for retries?